src/main/java/school/hei/haapi/endpoint/rest/controller/StudentFileController.java

```diff
@@ -0,0 +1,21 @@
+package school.hei.haapi.endpoint.rest.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import school.hei.haapi.service.StudentFileService;
+
+@RestController
+@AllArgsConstructor
+public class StudentFileController {
+  private final StudentFileService fileService;
+
+  @GetMapping(
+      value = "/students/{id}/scholarship_certificate/raw",
+      produces = {MediaType.APPLICATION_PDF_VALUE})
```

Comment:
try using a single value instead of an array as "produces" value

Reply:
i don't really understand can you give me some example ?

Reply:
produces = APPLICATION_PDF_VALUE